api/handlers.go

```diff
@@ -742,6 +743,10 @@ func (api *API) userPUT(u *database.User, w http.ResponseWriter, req *http.Reque
 		changedEmail = true
 	}
 
+	if api.staticDeps.Disrupt("DependencyUserPutMongoDelay") && fastrand.Intn(2) == 0 {
```

Comment:
kind of strange to see a `Disrupt` in combination with a chance factor, how does this not introduce NDFs?

Reply:
Honestly, I can't say. I don't it will. At worst, I guess it might create a false positive in which we break the retry mechanic but the test still passes.

This is a suggestion Matt did twice and I accepted. I'm not sure what it adds to the test but it shouldn't be an NDF.
See https://github.com/SkynetLabs/skynet-accounts/pull/224#discussion_r908836955

Reply:
Thinking about this more, I don't see a reason to add this randomness. I'll remove it.

@MSevey we can talk about it again if you think that we need it. I really don't see how it's helpful but maybe I'm just missing the point.